core/src/main/scala/package.scala

```diff
@@ -48,6 +48,7 @@ package object funnel {
     def append(a: A, b: => A) = G.plus(a, b)
   }
 
-  implicit val dontUseTheDefaultStrategy: scalaz.concurrent.Strategy = null
-  implicit val theDefaultStrategyCausesProblems: scalaz.concurrent.Strategy = null
+  // This doesn't work in 2.11; these are at the highest priority within the package.
+  // implicit val dontUseTheDefaultStrategy: scalaz.concurrent.Strategy = null
+  // implicit val theDefaultStrategyCausesProblems: scalaz.concurrent.Strategy = null
 }
```

Comment:
I think that's rather the point, that we don't want to use implicit Strategies in Funnel. No?